plugins/modules/sops_encrypt.py

```diff
@@ -16,7 +16,7 @@
 short_description: Encrypt data with sops
 version_added: '0.1.0'
 description:
-  - Allows to encrypt binary data (Base64 encoded), text data, JSON or YAML data with sops.
+  - Allows to encrypt binary data (Base64 encoded), text data, JSON or YAML data with sops. A binary executable C(sops) must exist in C($PATH).
```

Comment:
Thanks for your contribution!

How about documenting this in `requirements:` in [plugins/doc_fragments/sops.py](https://github.com/ansible-collections/community.sops/blob/main/plugins/doc_fragments/sops.py#L12-L83) instead? It should also reference to `O(sops_binary)`, which allows you to point the modules/plugins to a specific `sops` binary (which then doesn't have to be in `$PATH`).

Reply:
done in #174. I am closing this one.